pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -325,6 +325,10 @@ type ClusterCephxConfig struct {
 	// Daemon configures CephX key settings for local Ceph daemons managed by Rook and part of the
 	// Ceph cluster. Daemon CephX keys can be rotated without affecting client connections.
 	Daemon CephxConfig `json:"daemon,omitempty"`
+	// RBDMirrorPeer configures CephX key settings of the `rbd-mirror-peer` user that is used for creating
+	// bootstrap peer token used connect peer clusters. Rotating the `rbd-mirror-peer` user key will update
+	// the mirror peer token.
```

Comment:
This is a similar note to one proposed for CSI that should serve as a good warning to users not to do this without thinking

```suggestion
	// the mirror peer token.
	// Rotation will affect any existing peers connected to this cluster, so take care when exercising this option.
```